inactive-proposals.md

```diff
@@ -17,7 +17,8 @@ Inactive proposals are proposals that at one point were presented to the committ
 | [SIMD.JS - SIMD APIs][simd]                                          | Peter Jensen, Yehuda Katz            | [Stage 1][simd-notes]: Start with SIMD in WASM; implementations not pursuing SIMD.js for now.
 | [Updates to Tail Calls to include an explicit syntactic opt-in][ptc] | Brian Terlson & Eric Faust           | Inactive |
 | [Object.shallowEqual][shallow-equal]                                 | Sebastian Markbage                   | Withdrawn |
-
+| [`%constructor%.construct`][construct]                               | James M Snell                        | Withdrawn |
+| [`Builtins.typeOf()` and `Builtins.is()`][is-types]                  | James M Snell                        | Withdrawn: can be solved in other ways |
```

Comment:
Can you elaborate how it can be solved in other ways? I'm still very interested in this proposal, and I'm still unaware of any way to solve it for all builtins in a consistent way, and that doesn't require try/catch.

Reply:
The solution is imperfect and relies on the to string tags as has been suggested by others. If we're going to move this forward, we need to come up with some better solid reasons why the other solution doesn't work :-) ... we can discuss further in the proposal repo and we can add it back to the stage-0 proposals once we have.

Reply:
I'm still not clear on how it's imperfect, the perfect solution is "toStringTag, but brand-checking functions instead of strings", and the reasons that toStringTag doesn't work is because it's forgeable in both directions.